Assessment-April/Pset5_odd.py

```diff
@@ -0,0 +1,25 @@
+#Write a function such that consumes an int as its argument returns True only if the following conditions are met:
+
+# Conditions are
+# odd number of digits
+# has a zero in the center ,but nowhere else
+
+
+def check_num(num):
+    """
+    Returns True if the input integer has an odd number of digits and has a zero in the center,
+    but nowhere else. Returns False otherwise.
+    """
+    num_str = str(num)
```

Comment:
# This line converts the integer `num` into a string `num_str`. We do this to check whether the number of digits in `num` is odd or even, and to use string operations to check if the zero is in the center and to check if there is any other zero in the rest of the digits